api/peering.go

```diff
@@ -68,6 +70,8 @@ type PeeringGenerateTokenRequest struct {
 	Partition  string `json:"Partition,omitempty"`
 	Datacenter string
 	Token      string
+	// Meta is a mapping of some string value to any other string value
+	Meta map[string]string
```

Comment:
```suggestion
	Meta map[string]string `json:",omitempty"`
```

Reply:
Hm, could I ask why add the `omitempty`? would this be ent specific? I think in a previous PR, I understood from Chris that we only add `\`json:",omitempty"\`` for ENT things like namespaces or partitions

Reply:
You add it for anything that may be empty that you don't want to send via json to the other side as empty.